Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine
```

Comment:
Ya, if you use scratch you have to add *some* executable for it to run. With the autoland-ui we hit this problem, and just went with a small base image like you're doing (I think we might have used busybox though, as it's even smaller)